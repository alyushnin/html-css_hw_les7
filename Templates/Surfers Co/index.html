<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Surfers Co</title>
	
</head>
<body>
	
</body>
</html>