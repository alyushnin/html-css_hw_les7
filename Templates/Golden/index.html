<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="style.css">
	<link rel="stylesheet" href="fonts/Wisdom Script AJ.otf">
</head>
<body>
	<div class="wrapper">
		<header class="header">
			<div class="container">
				<div class="logo">
					<a href="#">Golden</a>
					<ul class="nav">
						<li><a href="#">Home</a></li>
						<li><a href="#">Services</a></li>
						<li><a href="#">Portfolio</a></li>
						<li><a href="#">About</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</div>
			</div>
		</header>
		<div class="home">
			<div class="container">
				<h3>
					Welcome To Our Studio!
				</h3>
				<h1>
					Itâ€™s nice to Meet you
				</h1>
				<button class="home__button">Tell Me More</button>
			</div>
		</div>
	</div>
	
</body>
</html>